src/utils/dataPresentationHelper.ts

```diff
@@ -14,3 +14,16 @@ export const getTokenExplorerLink = (
 export const getTxExplorerLink = (network: string, address: string): string => {
   return `${EXPLORER_MAP[network] || 'https://etherscan.io'}/tx/${address}`
 }
+
+export function formatAddress(
```

Comment:
I think we have a methods folder, you can put this there. 

All other formatting methods are there. 

maybe we could use a better name than dataPresentationHelper.

Reply:
ahh, didnt know it .. yes exaqctly, i wouldnt figure out that `methods` file is related to data presentation methods, imo we should rename the file to be more explicit about its content